dev

```diff
@@ -58,6 +58,11 @@ else
   COMPOSE="$COMPOSE -f docker-compose.dev.yml"
 fi
 
+if test -e "docker-compose.override.yml"; then
```

Comment:
Check to see if the `docker-compose.override.yml` file exists. If so, attach it to the list of docker-compose files.